src/python/bot/tasks/minimize_task.py

```diff
@@ -1395,8 +1422,14 @@ def minimize_file(file_path,
 
   # Specialized minimization strategy for javascript.
   if file_path.endswith('.js'):
-    return do_js_minimization(test_function, get_temp_file, data, deadline,
-                              threads, cleanup_interval, delete_temp_files)
+    #New Antlr-Tokenized minimization disabled until happy with behavior
```

Comment:
+1 to the Abhishek's comment above, just delete all these commented lines for now

Reply:
Done